src/newspack-ui/scss/elements/forms/_buttons.scss

```
@@ -5,20 +5,22 @@
 		border: 0;
 		border-radius: var(--newspack-ui-border-radius-m);
 		color: var(--newspack-ui-color-neutral-90);
+		cursor: pointer;
 		display: inline-flex;
 		font-family: var(--newspack-ui-font-family);
 		font-size: var(--newspack-ui-font-size-s);
 		font-weight: 600;
-		letter-spacing: initial;
-		text-transform: none;
-		gap: var(--newspack-ui-spacer-2);
+		gap: calc(var(--newspack-ui-spacer-base) / 2);
```

```suggestion
		gap: calc(var(--newspack-ui-spacer-base) / 2);
		letter-spacing: initial;
		text-transform: none;
```

These two styles should stay -- they seem redundant, but were to override some common Custom CSS interference we saw when testing clone sites 😅